### PR TITLE
Build for canary 221

### PR DESCRIPTION
**Pull request by stevemessick**

### CODE REVIEW

File: flutter-studio/src/io/flutter/actions/OpenFlutterNewsSidePanelAction.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ public void toolWindowRegistered(@NotNull String id) {
 
     @Override
     public void toolWindowUnregistered(@NotNull String id, @NotNull ToolWindow toolWindow) {
-      if (id.equals(OpenAssistSidePanelAction.TOOL_WINDOW_TITLE)) {
+      if (id.equals("Assistant:")) {
```
Comment (helin24) [inline flutter-studio/src/io/flutter/actions/OpenFlutterNewsSidePanelAction.java:127]: Do you want to make this a constant too?

File: flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java [hunk 2]
```diff
@@ -257,7 +257,7 @@ public void syncSkipped(@NotNull Project project) {
     };
 
     GradleSyncInvoker.Request request = new GradleSyncInvoker.Request(TRIGGER_PROJECT_MODIFIED);
-    request.runInBackground = true;
+    //request.runInBackground = true;
```
Comment (helin24) [inline flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java:260]: Do you intend to keep this?

**Review by helin24 — APPROVED** [review]

File: flutter-studio/src/io/flutter/actions/OpenFlutterNewsSidePanelAction.java (hunk 1 above)
Comment (stevemessick) [reply flutter-studio/src/io/flutter/actions/OpenFlutterNewsSidePanelAction.java:127]: No, this code isn't even used.

File: flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java (hunk 2 above)
Comment (stevemessick) [reply flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java:260]: Yes, I *think* I need to add the ability to remove the comment for all AS builds other than canary.

**Pull request merged**